src/page-modules/assistant/__tests__/assistant.test.tsx

```diff
@@ -33,6 +33,11 @@ import {
   getAssistantTripIfCached,
 } from '../server/trip-cache';
 import { FromToTripQuery } from '../types';
+import {
+  getHoursDifferenceFromCET,
+  getLastSundayOfMonthAndSetTime,
+  isDaylightSavingTime,
+} from '@atb/utils/date';
 
```

Comment:
Not to be difficult but these tests should be located in `utils/date` directory 🤔

Reply:
I suspected that these tests might be moved😅

Reply:
Should also move date.ts into `utils/date`? In that case, quite a few import statements to retrieve functions from date.ts will change, but that's the structure of the repo, right?

Reply:
I didn't check where date.ts was located. But date.ts can remain inside `utils` but you can create `utils/__tests__/date.test.ts` and move the tests over there. The point is to co-locate tests with the implementation.